test/test_codegen.py

```diff
@@ -127,7 +143,9 @@ def test_get_thermo_properties(mechname):
         keq_err = np.abs((keq_pm - keq_ct) / keq_ct).max()
```

Comment:
This propagated from kinetics, but my PR has a comment by @inducer to use `np.linalg.norm` instead.

Reply:
The thing I absolutely hate about the np.linalg norm is that it processes the entire vector and erases any component-specific information.   Typically, when doing the numerical debugging - I like to see the error in each component separately because it can help guide one to the problem(s).  So far I have been unsuccessful at arguing for code that allows component-by-component inspection; so changed.